modules/e2ee/ManagedKeyHandler.ts

```diff
@@ -1,30 +1,27 @@
 /// <reference types="node" />
 
-import { getLogger } from "@jitsi/logger";
 import browser from "../browser";
 import JitsiLocalTrack from "../RTC/JitsiLocalTrack";
 import JingleSessionPC from "../xmpp/JingleSessionPC";
 import TraceablePeerConnection from "../RTC/TraceablePeerConnection";
-import { CustomRTCRtpReceiver, CustomRTCRtpSender } from "./E2EEContext";
+import E2EEContext, { CustomRTCRtpReceiver, CustomRTCRtpSender } from "./E2EEContext";
 
 import * as JitsiConferenceEvents from "../../JitsiConferenceEvents";
 
 import Listenable from "../util/Listenable";
 import { OlmAdapter } from "./OlmAdapter";
 import JitsiConference from "../../JitsiConference";
-import E2EEContext from "./E2EEContext";
 import RTCEvents from "../../service/RTC/RTCEvents";
 import { generateEmojiSas } from "./SAS";
 
-const logger = getLogger(__filename);
-
 /**
  * This module integrates {@link E2EEContext} with {@link OlmAdapter} in order to distribute the keys for encryption.
  */
 export class ManagedKeyHandler extends Listenable {
     conference: JitsiConference;
     e2eeCtx: E2EEContext;
     enabled: boolean;
+    init: any;
```

Comment:
Avoid using the Any type if possible, it seems that initSessions returns an array of promises